OpenSim/Tools/Test/testSerializeOpenSimObjects.cpp

```diff
@@ -82,7 +82,9 @@ int main()
             try {
                 testModel.addModelComponent(ModelComponent::safeDownCast(randClone));
             } //Ignore the validity of the property values
-            catch (const SimTK::Exception::ErrorCheck&) {
+            // TODO this should specifically handle "InvalidPropertyValue" exceptions
+            // once we have that in place.
+            catch (const std::exception&) {
```

Comment:
testSerializeOpenSimObjects fills all string type properties with a generic "ABCXYZ" string which is invalid for a quadrant name. Then was "activated" because I know resolve the quadrant at `finalizeFromProperties` instead of side-effect of `connect()`